app/react-native/src/server/config/babel.prod.js

```diff
@@ -1,33 +1,21 @@
-/**
- * Copyright (c) 2015-present, Facebook, Inc.
- * All rights reserved.
- *
- * This source code is licensed under the BSD-style license found in the
- * LICENSE file in the root directory of this source tree. An additional grant
- * of patent rights can be found in the PATENTS file in the same directory.
- */
-
 module.exports = {
   // Don't try to find .babelrc because we want to force this configuration.
   babelrc: false,
   presets: [
-    // let, const, destructuring, classes, modules
-    require.resolve('babel-preset-es2015'),
-    // exponentiation
-    require.resolve('babel-preset-es2016'),
-    // JSX, Flow
+    [
+      require.resolve('babel-preset-env'),
+      {
+        modules: false,
+      },
+    ],
     require.resolve('babel-preset-react'),
+    require.resolve('babel-preset-minify'),
   ],
   plugins: [
-    // function x(a, b, c,) { }
```

Comment:
That's because we're not maintaining/documenting babel..

It's just a `.babelrc`. would you expect us to document/instruct people how to config that?